flink-runtime/src/test/java/org/apache/flink/runtime/scheduler/adaptive/AdaptiveSchedulerTest.java

```diff
@@ -1783,56 +1778,6 @@ void testStopWithSavepointFailsInIllegalState() throws Exception {
                 .withCauseInstanceOf(CheckpointException.class);
     }
 
-    @Test
-    void testSavepointFailsWhenBlockingEdgeExists() throws Exception {
```

Comment:
do you know why / when this was introduced? it indeed doesn't seem to be related AS; is this tested somewhere else?

Reply:
This was added in FLINK-34371 and is covered by tests in the DefaultScheduler:

https://github.com/apache/flink/commit/d4e0084649c019c536ee1e44bab15c8eca01bf13#diff-b4bc1cd606feb86850a18371520b5dd63d02090b8567fdf80730d1d7dd6e693d